hack/ginkgo-e2e.sh

```diff
@@ -102,6 +102,7 @@ export PATH=$(dirname "${e2e_test}"):"${PATH}"
   --repo-root="${KUBE_ROOT}" \
   --node-instance-group="${NODE_INSTANCE_GROUP:-}" \
   --prefix="${KUBE_GCE_INSTANCE_PREFIX:-e2e}" \
+  --os-distro="${OS_DISTRIBUTION:-debian}" \
```

Comment:
If so, I think we need to use KUBE_OS_DISTRIBUTION and export it in our trusty-based jenkins jobs.